src/services/aws.js

```diff
@@ -0,0 +1,30 @@
+import AWS from 'aws-sdk/global'
+import 'aws-sdk/clients/dynamodb'
+
+const ROLE_ARN = 'arn:aws:iam::407461997746:role/vueaws'
+
+let AwsDocClient = null
+let awsDocClientResolve = null
+const awsDocClientInit = new Promise((resolve) => awsDocClientResolve = resolve)
```

Comment:
Yes, that syntax is new to me. I need to read up about "Promises". I am guessing it's like a bash named pipe.

So thinks get triggered only once resolve is called from auth, right?